### ISSUE TITLE
Generate command fails with existing rep

### ISSUE THREAD

**Issue by clemsos** [ludobox/ludobox#3]
```
venv)╭─clemsos@miner  ~/Dev/ludobox/ludobox-ui ‹ruby-2.2.1› ‹master› 
╰─$ python ludocore.py generate
Create games directory: FAIL >> '/home/clemsos/Dev/ludobox/ludobox-ui/static/games' already exist impossible to create it. Try to use './ludocore.py clean' to remove it.
Traceback (most recent call last):
  File "ludocore.py", line 1167, in <module>
    main()
  File "ludocore.py", line 1163, in main
    return args.func(**vars(args))  # We use `vars` to convert args to a dict
  File "ludocore.py", line 750, in generate_all
    for path in sorted(os.listdir(input_dir)):
OSError: [Errno 2] No such file or directory: '/home/clemsos/Dev/ludobox/ludobox-ui/data'

**Comment by clemsos** [comment]
```./ludocore.py clean ``` the reproduces #1

**Comment by clemsos** [comment]
fixed in #6
